Scripts/Items/Consumables/WaterBarrel.cs

```diff
@@ -12,7 +13,7 @@ public WaterBarrel()
 
         [Constructable]
         public WaterBarrel(bool filled)
```

Comment:
They will be marked as non-movable when they are placed by filling the fillable barrel.